bluebrain/repo-bluebrain/packages/spatial-index/package.py

```diff
@@ -13,6 +13,7 @@ class SpatialIndex(PythonPackage):
     url      = "git@bbpgitlab.epfl.ch:hpc/SpatialIndex.git"
 
     version('develop', branch='main', submodules=True)
+    version('0.5.2-rc2', commit="1a5a78c41d449582a843bcced41aac11bb96a2d0", submodules=True)
     version('0.5.2-rc1', commit="796da683101e1fd3916644064703c9524a0e1e99", submodules=True)
```

Comment:
Just a note: if we are certain that `rc1` won't be required in future for production usage (i.e. was a temporary one), it's fine to remove it.

Reply:
Thank you! I intended to remove it once we release. Good to know removing is safe with spack recipes.